### PR TITLE
Enable 7940hs runner

### PR DESCRIPTION
**Pull request by fifield**
try switching workflow to 7940hs runner

### CODE REVIEW

**Comment by makslevental** [comment]
@fifield can you please just duplicate the file instead of trying to change it ~ `buildAndTestAieToolsSelfHosted.yml`

File: .github/workflows/buildAndTestRyzenAI.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+              -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+              -DCMAKE_INSTALL_PREFIX=install
+
+            ninja install
+            cp /usr/lib/firmware/amdaie/1502/1x4.xclbin install/data/1x4.xclbin
+
+            sed -i.bak 's/-sv --timeout 600/-sv --timeout 600 -j12/g' ../test/CMakeLists.txt
+            PATH=/opt/Xilinx/Vitis/2023.2/aietools/bin:$PATH ninja check-aie
+
+            rm -rf build
```
Comment (keryell) [inline .github/workflows/buildAndTestRyzenAI.yml:86]: Curious the ReviewDog do not catch the missing EOL. :-) @makslevental

Comment (makslevental) [reply .github/workflows/buildAndTestRyzenAI.yml:86]: i didn't setup a linter for the yaml files

File: test/ipu-xrt/lit.local.cfg [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 config.suffixes = ['.lit']
 config.unsupported = []
+config.suffixes = ['.lit']
```
Comment (stephenneuendorffer) [inline test/ipu-xrt/lit.local.cfg:10]: extra?

File: .github/workflows/buildAndTestRyzenAI.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+              -DCMAKE_EXE_LINKER_FLAGS_INIT="-fuse-ld=lld" \
+              -DCMAKE_MODULE_LINKER_FLAGS_INIT="-fuse-ld=lld" \
+              -DCMAKE_SHARED_LINKER_FLAGS_INIT="-fuse-ld=lld" \
+              -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+              -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+              -DCMAKE_INSTALL_PREFIX=install
+
+            ninja install
+            cp /usr/lib/firmware/amdaie/1502/1x4.xclbin data/1x4.xclbin
+            cp /usr/lib/firmware/amdaie/1502/1x4.xclbin install/data/1x4.xclbin
```
Comment (stephenneuendorffer) [inline .github/workflows/buildAndTestRyzenAI.yml:67]: no longer necessary

File: test/ipu-xrt/lit.local.cfg (hunk 2 above)
Comment (fifield) [reply test/ipu-xrt/lit.local.cfg]: fixed

File: .github/workflows/buildAndTestRyzenAI.yml (hunk 3 above)
Comment (fifield) [reply .github/workflows/buildAndTestRyzenAI.yml]: thanks!

**Comment by fifield** [comment]
@makslevental do you know what's going on here in `Lint and Format / Check`?

```
Saved working directory and index state WIP on (no branch): 0cca4cb Merge 8d30442dfc8f01879147dd710239acaf2c60f856 into 5ebaf21c1d2a22aa342f9524b755cb75a2ca3e96
cmake/modulesXilinx:1:-Subproject commit 44b1881a16c27e6932744650055d1179e5cd8442
cmake/modulesXilinx:1:+Subproject commit 44b1881a16c27e6932744650055d1179e5cd8442-dirty
reviewdog: input data has violations
Error: Process completed with exit code 1.
```
https://github.com/Xilinx/mlir-aie/actions/runs/7134684317/job/19429982287?pr=812#step:11:216

**Comment by makslevental** [comment]
> https://github.com/Xilinx/mlir-aie/actions/runs/7134684317/job/19429982287?pr=812#step:11:216

Unfortunately I do not - not sure why it's messing with cmake modules at all. Lemme try to "turn it off and turn it back on again"

**Comment by makslevental** [comment]
> https://github.com/Xilinx/mlir-aie/actions/runs/7134684317/job/19429982287?pr=812#step:11:216

I believe the error is (somehow) because the submodule ref is old on this branch. I turned off `fail_on_error` for now (which I don't really like) but let's see.

**Pull request merged**